dom/data/core.js

```diff
@@ -1,77 +1,9 @@
 'use strict';
 
-var isEmptyObject = require("../../js/is-empty-object/is-empty-object");
+var canDev = require("can-util/js/dev/dev");
 
-var data = {};
-var expando = "can" + new Date();
-var uuid = 0;
+ //!steal-remove-start
+ canDev.warn('dom/data/core is deprecated; please use can-dom-data-state instead: https://github.com/canjs/can-dom-data-state');
```

Comment:
This is not a public API, so does not need a deprecation notice. You should just delete it.